Clarify language around operating system updates (#3690)
theforeman/foreman-documentation: guides/common/modules/con_updating-project-to-next-patch-version.adoc

You can update your {ProjectServer} and {SmartProxyServer} to a new patch release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1, by using the {Project} maintain tool.
The patch releases are non-disruptive to your operating environment and often fast.

> {ProjectServer} or {SmartProxyServer} updates should always be done using `{foreman-maintain} update`.
> Operating systems updates should be performed the same way.

Reviewer: ```suggestion
Always update {ProjectServer}, {SmartProxyServer}, or the operating system by using `{foreman-maintain} update`.
```
"Should" is very ambiguous and we should avoid it altogether. Either "do" or "don't".
How does it sound if we simplify it?
